src/constants/user.request.error.constants.js

```diff
@@ -0,0 +1,13 @@
+export const userErrorConstants = {
+  SERVER_ERROR: 'Error connecting with server. Please try again later.',
+  LOGIN_ERROR_CREDENTIALS:
+    'Unable to login with provided credentials. Please check email and password.',
+  RESET_PWD_INVALID_TOKEN_ERROR:
+    'An error ocurred. Please go to the "Forgot your password?" link to start over the password recovery.',
+  GET_TARGETS_ERROR:
+    'An error ocurred while getting your targets. Please refresh or try again later',
+  GET_MATCHES_ERROR:
```

Comment:
as you are using internationalization, would'n these constants be placed in spanish too?

Reply:
What do you mean? to use intl here?